openssl-dynamic/src/main/c/ssl.c

```diff
@@ -678,21 +935,34 @@ TCN_IMPLEMENT_CALL(jint, SSL, getError)(TCN_STDARGS,
     return SSL_get_error(ssl_, ret);
 }
 
-// How much did SSL write into this BIO?
-TCN_IMPLEMENT_CALL(jint /* nbytes */, SSL, pendingWrittenBytesInBIO)(TCN_STDARGS,
-                                                                     jlong bio /* BIO * */) {
-    BIO *b = J2P(bio, BIO *);
+TCN_IMPLEMENT_CALL(jint /* nbytes */, SSL, bioCtrlPending)(TCN_STDARGS, jlong bio /* BIO* */) {
+    BIO* b = J2P(bio, BIO*);
 
     if (b == NULL) {
         tcn_ThrowException(e, "bio is null");
         return 0;
     }
 
-    UNREFERENCED_STDARGS;
-
     return BIO_ctrl_pending(b);
 }
 
+#ifdef OPENSSL_IS_BORINGSSL
```

Comment:
`BIO_wpending`? I'm... not sure why OpenSSL has two spellings for the same thing. (Or why we have both spellings for `BIO_pending` but only one for `BIO_wpending`. I guess nothing used `BIO_ctrl_wpending` so we didn't bother?)